packages/twenty-front/src/modules/context-store/components/MainContextStoreProviderEffect.tsx

```diff
@@ -18,9 +19,11 @@ export const MainContextStoreProviderEffect = ({
   pageName,
 }: {
   viewId?: string;
-  objectMetadataItem: ObjectMetadataItem;
-  pageName: string;
+  objectMetadataItem?: ObjectMetadataItem;
```

Comment:
pageName should always be defined
objectMetadataItem can be undefined indeed in Settings page case